contracts/ark_starknet/src/executor.cairo

```diff
@@ -181,6 +225,11 @@ mod executor {
             };
 
             let vinfo = CreateOrderInfo { order: order.clone() };
+            _verify_create_order(@self, @vinfo);
+
+            let order_hash = order.compute_order_hash();
+            let order_info = order.into();
+            self.orders.write(order_hash, order_info);
```

Comment:
i would add the original creator from the caller to be sure here so we can find it back i don't think we need the full order wdyt ?

just a key value:
order_hash: order_creator

Reply:
Creator of the order added in struct.